bog_app/config/routes.rb

```diff
@@ -0,0 +1,6 @@
+Rails.application.routes.draw do
+  root "creatures#index"
+
+  resources :creatures, only: [:index, :new, :create, :show, :edit, :update]
```

Comment:
Nice timesaver! When you're using resources, you can run the terminal command `rake routes` to see the paths and controller actions all written out. When you start CRUDing multiple models, it will be even more important to understand exactly which paths are being generated by these shortcuts.